templates/etc/openldap/slapd.conf.erb

```diff
@@ -79,7 +79,10 @@ end
 <% if @use_tls -%>
 TLSCertificateFile <%= @app_pki_cert %>
 TLSCertificateKeyFile <%= @app_pki_key %>
-TLSCipherSuite <%= @tls_cipher_suite.join(':') %>
+<% if @_tls_protocol_min -%>
```

Comment:
convention is to 'indent' the if, to show nesting.  So two more spaces between '<%' and 'if'.